### PR TITLE
To show Micro and Marco into Product Model.

### PR DESCRIPTION
**Pull request by lligato**

### CODE REVIEW

File: src/Models/ProductFeed.php [hunk 1]
```diff
@@ -343,6 +346,8 @@ public static function search($filter = array())
             $products[] = $p;
         }
 
+        //dd($products);
```
Comment (Giako) [inline src/Models/ProductFeed.php]: Is it possible to remove this commented out log?

**Review by Giako — CHANGES_REQUESTED** [review]
Just a little clean-up and it should be fine

File: src/Models/ProductFeed.php (hunk 1 above)
Comment (lligato) [reply src/Models/ProductFeed.php]: yes!

**Review by Giako — APPROVED** [review]
LGTM 👍

**Pull request merged**